src/models/case-audit.ts

```diff
@@ -0,0 +1,46 @@
+import { BuildOptions, DataTypes, Model, Sequelize } from 'sequelize';
+
+interface CaseAuditAttributes {
+  accountSid: string;
+  caseId: number;
+  newValue: string;
+  previousValue: string;
+  twilioWorkerId: string;
+};
+
+export interface CaseAuditModel extends Model<CaseAuditAttributes>, CaseAuditAttributes {}
+
+export class CaseAudit extends Model<CaseAuditModel, CaseAuditAttributes> { }
+
+export type CaseAuditStatic = typeof Model & {
+  new (values?: object, options?: BuildOptions): CaseAuditModel;
+};
+
+export function CaseAuditFactory (sequelize: Sequelize): CaseAuditStatic {
+  return <CaseAuditStatic>sequelize.define('CaseAudit', {
+    accountSid: { type: DataTypes.STRING },
+    caseId: { type: DataTypes.NUMBER },
+    newValue: { type: DataTypes.STRING },
+    previousValue:  { type: DataTypes.STRING },
+    twilioWorkerId: { type: DataTypes.STRING }, 
+  })
+};
```

Comment:
Yes, this class looks a little bit more complicated than what we would expect. Sequelize really doesn't help.
But does it work?

Reply:
It compiles 😆 , but It's throwing some runtime errors since I wasn't able to load the models dynamically (ToDo comment on .src/models/index.ts), Also, it seems to be too much boilerplate to add to our models. We need to figure out a better way.